library/blivet.py

```diff
@@ -1401,14 +1407,65 @@ def activate_swaps(b, pools, volumes):
+                                                          encryption_key_size=dict(type='int'),
+                                                          encryption_luks_version=dict(type='str'),
+                                                          encryption_password=dict(type='str'),
+                                                          fs_create_options=dict(type='str'),
+                                                          fs_label=dict(type='str', default=''),
+                                                          fs_type=dict(type='str'),
+                                                          mount_point=dict(type='str'),
+                                                          name=dict(type='str'),
+                                                          size=dict(type='str'),
+                                                          state=dict(type='str', default='present', choices=['present', 'absent']),
+                                                          type=dict(type='str'),
+                                                          vdo_pool_size=dict(type='str'))))),
+        volumes=dict(type='list', elements='dict',
+                     options=dict(disks=dict(type='list'),
+                                  encryption=dict(type='bool'),
+                                  encryption_cipher=dict(type='str'),
+                                  encryption_key=dict(type='str'),
+                                  encryption_key_size=dict(type='int'),
+                                  encryption_luks_version=dict(type='str'),
+                                  encryption_password=dict(type='str'),
+                                  fs_create_options=dict(type='str'),
+                                  fs_label=dict(type='str', default=''),
+                                  fs_type=dict(type='str'),
+                                  mount_point=dict(type='str'),
+                                  name=dict(type='str'),
+                                  raid_level=dict(type='str'),
+                                  raid_device_count=dict(type='int'),
+                                  raid_spare_count=dict(type='int'),
+                                  raid_metadata_version=dict(type='str'),
+                                  size=dict(type='str'),
+                                  state=dict(type='str', default='present', choices=['present', 'absent']),
+                                  type=dict(type='str'))),
         packages_only=dict(type='bool', required=False, default=False),
         disklabel_type=dict(type='str', required=False, default=None),
         safe_mode=dict(type='bool', required=False, default=True),
         pool_defaults=dict(type='dict', required=False),
         volume_defaults=dict(type='dict', required=False),
-        use_partitions=dict(type='bool', required=False, default=True),
+        use_partitions=dict(type='bool', required=False),
```

Comment:
This means the default value for `use_partitions` is `None` - and `if use_partitions:` will return False if `use_partitions` is either `False` or `None` - do you need to distinguish the case where the user does not set `use_partitions` vs. the user explicitly sets `use_partitions: False`?